packages/storybook/stories/va-checkbox-group-uswds.stories.tsx

```diff
@@ -413,3 +459,15 @@ export const FormsPatternMultiple = FormsPatternMultipleTemplate.bind(null);
 FormsPatternMultiple.args = {
   ...defaultArgs,
 };
+
+export const ToggleErrorState = ToggleErrorStateTemplate.bind(null);
+ToggleErrorState.args = {
+  focusEl: null,
+};
+ToggleErrorState.argTypes = {
+  focusEl: {
+    name: 'Element to focus on error toggle',
```

Comment:
Was it considered to update the [existing error stories](https://4616-form-error-messages--65a6e2ed2314f7b8f98609d8.chromatic.com/?path=/story/uswds-va-text-input--error) instead of adding another story for each component? 

I'm wondering this for two reasons:

1. Consolidating error behavior in one place might be easier to find.
2. Each story creates an additional Chromatic snapshot and if an error state story already exists, maybe we can just reuse that and in the process save ourselves from additional Chromatic story charges. I definitely don't want us to make documentation decisions based on that but maybe worth considering.

Reply:
@amyleadem Very cool! Quick question, when I select `#input-error-message` and toggle the error state, should the red text "There has been an error" be focused? I'm not seeing that happening if so. The others work fine though!

Reply:
It looks like it does appropriately move focus, but the focus indicator isn't visible.  If you turn on VoiceOver, you should hear it announce the the error message and VO will provide its own visual indicator. As a follow on, we'll need to look into why the focus indicator isn't visible there.